src/Repository/QuestionTemplateRepository.php

```diff
@@ -1,12 +1,57 @@
 <?php
 
-
 namespace QuizApp\Repository;
 
-
 use ReallyOrm\Repository\AbstractRepository;
 
+/**
+ * Class QuestionTemplateRepository
+ * @package QuizApp\Repository
+ */
 class QuestionTemplateRepository extends AbstractRepository
 {
+    /**
+     * This method looks in the Database for questions with the text LIKE $text
+     * and from an offset specified returning the questions found
+     *
+     * the quantity of questions to be retuned is specified in the $size variable
+     *
+     * @param $text
+     * @param int $from
+     * @param int $size
+     * @return array
+     */
+    public function getQuestionsByText($text, int $from, int $size)
```

Comment:
Suggestion:

"Returns questions that contain the value of the $text argument in their `text` fields.
The match is performed using the LIKE comparison operator.
The result set is paginated.

@param string $text The query string
@param int $from The offset
@param int $size The size of the result set (LIMIT)
@return array
"